src/packages/@ncigdc/modern_components/ClinicalAnalysis/ContinuousAggregationQuery.js

```diff
@@ -191,47 +201,55 @@ const getContinuousAggs = ({
     }));
 };
 
+const ContinuousAggregationQuery = ({
+  aggData,
+  hits,
+  isLoading,
+  setId,
+  stats,
+  ...props
+}) => ( // TODO this loader doesn't belong here. Must fix the continuous variable card dependency
+  isLoading
+  ? <Loader />
+  : (
+    <ContinuousVariableCard
+      data={{
+        ...aggData,
+        hits,
+      }}
+      setId={setId}
+      stats={stats}
+      {...props}
+      />
+    )
+);
+
 export default compose(
+  setDisplayName('EnhancedContinuousAggregationQuery'),
   withState('aggData', 'setAggData', null),
   withState('isLoading', 'setIsLoading', true),
-  withProps({
-    updateData: async ({
+  withPropsOnChange(
```

Comment:
this change prevents the card from updating on switch cohort

Reply:
Reverted this :) thanks